locale/en/locale.po

```diff
@@ -159,13 +155,9 @@ msgstr ""
 
 msgid "plugins.generic.pidManager.pidinst.workflow.instructions"
 msgstr ""
-"Here you can add existing PIDINSTs to the publication. "
-"Use the upper table to look up the PIDINST that you want to add. Enter the DOI or title of the PIDINST. "
-"Start the search by clicking the search button and select one of the suggested PIDINSTs; autocomplete will assist you. "
-"You can select multiple PIDINSTs in one search and close the suggestions by clicking the 'X' button. "
-"Greyed-out suggestions have already been added. "
-"The lower table shows the available metadata for the PIDINSTs you added. "
-"These properties will also be displayed in the article view. You can manually add an PIDINST and edit the metadata values."
+"You can add PIDINSTs to a publication by searching with a DOI or title in the upper table, selecting from autocomplete sugesstions, and avoiding duplicated (greyed-out). "
```

Comment:
"selecting from autocomplete suggestions, and avoiding duplicates (greyed-out)."